packages/calypso-e2e/src/lib/pages/my-home-page.ts

```diff
@@ -11,10 +11,29 @@ import { Page } from 'playwright';
 const selectors = {
 	dashboard: '.customer-home__main',
 	statsCard: '.stats',
+	visitSiteButton: 'text=Visit site',
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/53323#discussion_r648149864. I can see there's an issue for it already, so that's awesome! :D https://github.com/Automattic/wp-calypso/issues/53556

Reply:
This isn't/wasn't in the scope of #53556, but do you think it should be?

This selector seems to be working fine as is, and while I don't discount that another `Visit site` text might show up later down the road - I don't think it would be attached to the DOM earlier than this specific button.

Reply:
I've added a narrowing selector in the most recent revision, it now reads `.customer-home__view-site-button >> text=Visit site`

Reply:
I might need to take back my suggestion here. Sorry for the confusion, I'm learning too! 😬  Just read the Good Practices for the PW Selectors section and it seems that avoiding non-user-facing attributes is actually [considered a good practice](https://playwright.dev/java/docs/selectors#prioritize-user-facing-attributes). Now that I think about it, it's been often an issue with Selenium errors that the test got broken because a class selector changed. Maybe PW has a more efficient mechanism for querying with text only and it doesn't really make sense to make those selectors more specific? What do you think - should we go back and drop the class selectors?

Reply:
Ah, I should have linked you that page! 

You are right that Selenium has been pretty brittle because of the selectors - mostly xpath, which I try to avoid, and CSS selectors. I just ran into this earlier as I was developing the `Media Edit` spec, since my personal WPCOM account seems to have a different image editor than what's available to the general public. If I had used user facing attributes most of this could have been avoided.

> Maybe PW has a more efficient mechanism for querying with text only and it doesn't really make sense to make those selectors more specific?

Playwright is indeed quite heavy on text selectors. For most things, text selectors alone function just fine, but as we've seen in the case of sidebars in #53566, sometimes the text selectors will match multiple things and the first match is not necessarily the one we want. Now, use of specific CSS selectors are often inevitable, but where possible we should try to use the simplest form possible.

Looping back to what you've said re: dropping class selectors - I think we should leave it is for now, as I do not anticipate the home dashboard changing anytime soon. We can revisit once tests begin failing. How is that?

Reply:
Sounds good. I guess it's about finding a way between sticking to good practices and making sure we're targeting the right element. A single classname shouldn't be that brittle 👍